mkosi/resources/mkosi.md

```diff
@@ -1925,6 +1925,21 @@ to build.
 
 Note that the minimum required Python version is 3.9.
 
+# Frequently Asked Questions (FAQ)
+
+- Why does `mkosi qemu` with KVM not work on Debian/Ubuntu?
+
+While other distributions are OK with allowing access to `/dev/kvm`, on
+Debian/Ubuntu this is only allowed for users in the `kvm` group. Because
+mkosi unshares a user namespace when running unprivileged, it loses
```

Comment:
…, even if the calling user was in the `kvm` group, when mkosi unshares the user namespace to run unprivileged, …